packages/fxa-settings/src/components/Settings/Page2faReplaceRecoveryCodes/index.tsx

```diff
@@ -174,15 +174,19 @@ export const Page2faReplaceRecoveryCodes = (_: RouteComponentProps) => {
                 type="button"
                 className="cta-neutral cta-base-p mx-2 flex-1"
                 onClick={goHome}
+                data-glean-id="two_step_auth_codes_cancel"
+                data-glean-type="replace"
               >
                 Cancel
               </button>
             </FtlMsg>
-            <FtlMsg id="recovery-key-continue-button">
+            <FtlMsg id="tfa-button-continue">
```

Comment:
I noticed there was a complaint in the console about a missing ID in the bundle.